core/src/main/scala/org/apache/spark/util/AccumulatorV2.scala

```diff
@@ -415,15 +415,20 @@ class DoubleAccumulator extends AccumulatorV2[jl.Double, jl.Double] {
 }
 
 
-class ListAccumulator[T] extends AccumulatorV2[T, java.util.List[T]] {
+/**
+ * An [[AccumulatorV2 accumulator]] for collecting a list of elements.
+ *
+ * @since 2.0.0
+ */
+class CollectionAccumulator[T] extends AccumulatorV2[T, java.util.List[T]] {
```

Comment:
Question: so this method still accumulates a list, despite its name (which does sound better). Can an accumulator guarantee an order to the items add to the list meaningfully? if not, then maybe it should be advertised as an accumulator of a Collection.